isos/appliance.sh

```diff
@@ -83,6 +83,14 @@ ln -s /etc/systemd/system/vic.target $(rootfs_dir $PKGDIR)/etc/systemd/system/mu
 rm -f $(rootfs_dir $PKGDIR)/etc/systemd/network/*
 cp ${DIR}/base/no-dhcp.network $(rootfs_dir $PKGDIR)/etc/systemd/network/
 
+#
+# Set up vicadmin user
+#
+
+chroot $(rootfs_dir $PKGDIR) groupadd -g 1000 vicadmin
```

Comment:
`chroot` just to add a line to the groups file sounds awkward.  How bout just using `echo >>`